repositories.bzl

```diff
@@ -1,7 +1,7 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-BORINGSSL_COMMIT = "9df0c47bc034d60d73d216cd0e090707b3fbea58"  # same as Envoy
```

Comment:
Update this to actually be the same as current Envoy. An update was needed to get a new enough boringssl to include ed25519 support.

https://github.com/envoyproxy/envoy/blob/master/bazel/repository_locations.bzl#L76

Reply:
Please add the date in the comment, just like envoy, e.g.
// 2020-03-16